R/compare_df_cols.R

```diff
@@ -241,18 +243,18 @@ compare_df_cols_same <- function(..., bind_method = c("bind_rows", "rbind"), ver
 #'
 #' @details For package developers, an S3 generic method can be written for
 #'   `describe_class()` for custom classes that may need more definition
-#'   than the default method.  This function is called by `compare_df_cols`.
+#'   than the default method.  This function is called by [compare_df_cols()].
 #'
 #' @param x The object to describe
 #' @param strict_description Should differing factor levels be treated
-#' as differences for the purposes of identifying mismatches?
-#' `strict_description = TRUE` is stricter and factors with different
-#' levels will be treated as different classes.  `FALSE` is more
-#' lenient: for class comparison purposes, the variable is just a "factor".
+#'   as differences for the purposes of identifying mismatches?
+#'   `strict_description = TRUE` is stricter and factors with different
+#'   levels will be treated as different classes.  `FALSE` is more
+#'   lenient: for class comparison purposes, the variable is just a "factor".
 #' @return A character scalar describing the class(es) of an object where if the
 #'   scalar will match, columns in a data.frame (or similar object) should bind
 #'   together without issue.
-#' @family Data frame type comparison
+#' @family data frame type comparison
```

Comment:
Using lower-case is better, because it will show up as "Other data frame type comparison" in man (feels more like a sentence than "Other Data frame type comparison" in my opinion

Reply:
That makes sense to me.